sdk/core/azure-core/azure/core/pipeline/transport/_aiohttp.py

```diff
@@ -332,7 +332,7 @@ async def __anext__(self):
                 if not self._decompressor:
                     import zlib
 
-                    zlib_mode = 16 + zlib.MAX_WBITS if enc == "gzip" else zlib.MAX_WBITS
+                    zlib_mode = 16 + zlib.MAX_WBITS if enc == "gzip" else -zlib.MAX_WBITS
```

Comment:
is the `else` 16 - zlib.MAX_WBITS` or just `-zlib.MAX_WBITS`. I think it's the latter but it's not the clearest

Reply:
It is just -zlib.MAX_WBITS